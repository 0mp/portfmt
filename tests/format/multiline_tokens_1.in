USES=	a b c d f e h z a   6 87  3 3 5 67 8 9 9  \
	345345 3453 \
	3 \
	sdf \
gsd

