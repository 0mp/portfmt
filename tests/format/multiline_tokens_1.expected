USES=		3 3453 345345 5 6 67 8 87 9 a b c d e f gsd h sdf z

